noodles/display/dumb_term.py

```diff
@@ -96,7 +98,10 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                 self.out << "\n" << "User interrupt detected, abnormal exit.\n"
                 return True
 
-            print("Internal error encountered. Contact the developers.")
+            logger.critical(
+                "Internal error encountered. Contact the developers.",
+                exc_info=exc_val,
+            )
```

Comment:
An example of a logging message where a traceback will be included.

Reply:
An example of such a tracebacked logging output from https://github.com/SCM-NV/nano-qmflows/runs/6536779272?check_suite_focus=true:
```
INFO     nanoqm:input_validation.py:191 path to basis added to cp2k settings
INFO     nanoqm:initialization.py:59 Using nano-qmflows version: 0.12.1 
INFO     nanoqm:initialization.py:60 nano-qmflows path is: /opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/nanoqm
INFO     nanoqm:initialization.py:61 Working directory is: /tmp/nanoqm
INFO     nanoqm:initialization.py:62 Data will be stored in HDF5 file: /tmp/pytest-of-runner/pytest-0/test_workflow_IPR0/Cd33Se33.hdf5
INFO     nanoqm:initialization.py:177 An initial Calculation will be computed as guess for the wave function
INFO     nanoqm:tools.py:22 Starting single point calculation.
INFO     nanoqm:initialization.py:59 Using nano-qmflows version: 0.12.1 
INFO     nanoqm:initialization.py:60 nano-qmflows path is: /opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/nanoqm
INFO     nanoqm:initialization.py:61 Working directory is: /tmp/nanoqm
INFO     nanoqm:initialization.py:62 Data will be stored in HDF5 file: /tmp/pytest-of-runner/pytest-0/test_workflow_IPR0/Cd33Se33.hdf5
INFO     nanoqm:initialization.py:177 An initial Calculation will be computed as guess for the wave function
INFO     nanoqm:workflow_single_points.py:41 starting!
INFO     nanoqm:components.py:114 point_0 has been scheduled
INFO     nanoqm:components.py:224 Total energy of point 0 is: -1832.3239105553077
ERROR    noodles:scheduler.py:105 Uncaught error running job 140216626697056
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/noodles/run/worker.py", line 34, in run_job
    result = node.apply()
  File "/opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/noodles/workflow/model.py", line 57, in apply
    return self.foo(*self.bound_args.args, **self.bound_args.kwargs)
  File "/opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/nanoqm/schedule/components.py", line 159, in store_molecular_orbitals
    save_orbitals_in_hdf5(mos, config, dict_input["job_name"])
  File "/opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/nanoqm/schedule/components.py", line 173, in save_orbitals_in_hdf5
    dump_orbitals_to_hdf5(mos, config, job_name)
  File "/opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/nanoqm/schedule/components.py", line 202, in dump_orbitals_to_hdf5
    store_arrays_in_hdf5(config.path_hdf5, path_property, array)
  File "/opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/nanoqm/common.py", line 288, in store_arrays_in_hdf5
    dset = f5.require_dataset(paths, shape=np.shape(
  File "/opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/h5py/_hl/group.py", line 239, in require_dataset
    raise TypeError("Shapes do not match (existing %s vs new %s)" % (dset.shape, shape))
TypeError: Shapes do not match (existing (80,) vs new (100,))
```